### PR TITLE
CRA typescript

### PR DESCRIPTION
**Pull request by dchasepdx**
react-scripts-ts was deprecated in favor of create react app supporting TS. So I migrated back to that and cleaned up some stuff that I broke in an earlier merge.

### CODE REVIEW

**Review by andybeers — COMMENTED** [review]
oooh nice, good catch

**Review by andybeers — APPROVED** [review]

File: src/App.tsx [hunk 1]
```diff
@@ -1,36 +1,20 @@
-/** @jsx jsx */
-import { Component, Fragment } from 'react'
-import { jsx } from '@emotion/core'
-import * as Hammer from 'hammerjs'
-import Info from './Info'
+import React, { Component } from 'react'
+import Hammer from 'hammerjs'
```
Comment (andybeers) [inline src/App.tsx:2]: Do these default imports have to be `import * as`?

Comment (dchasepdx) [reply src/App.tsx:2]: I'm not sure. What does `import * as` do? Is it a typescript thing to get a libraries types imported along with the default?

Comment (andybeers) [reply src/App.tsx:2]: Huh, great question -- this is running A-ok!

I was getting an error when running the old setup. I'll see if I can figure out what magic CRA is doing here

**Pull request merged**